### PR TITLE
Update public cache content

### PR DESCRIPTION
**Pull request by dominicfernando**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

Added new line about cacheable="false" in default.xml

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3710) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3710) before we can accept your contribution.<br/>

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ Add a `cacheable="false"` attribute to any block in your layout to disable cachi
 
 Magento disables page caching if at least one non-cacheable block is present in the layout.
 
+{: .bs-callout .bs-callout-warning }
+Caching will be disabled in all the pages of the site when `cacheable="false"` is used inside the `default.xml`.
```
Comment (jeff-matthews) [inline guides/v2.1/extension-dev-guide/cache/page-caching/public-content.md]:
```suggestion
Using `cacheable="false"` inside the `default.xml` file disables caching for all pages on the site.
```

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**